pubsub-adapter/src/test/kotlin/com/ably/Utils.kt

```diff
@@ -15,3 +18,29 @@ suspend fun waitFor(timeoutInMs: Long = 10_000, block: suspend () -> Boolean) {
     }
   }
 }
+
+fun createAblyRealtime(port: Int): AblyRealtime {
```

Comment:
Is it okay if we don't mark methods public explicitly 🤔. Or the check not applicable on tests?

Reply:
yeah, this rule applies for the main source sets only